dll/win32/KernelBase/kernelbase.spec

```diff
@@ -228,6 +230,7 @@
 @ stdcall EnumResourceLanguagesExW(long wstr wstr ptr long long long)
 @ stdcall EnumResourceNamesExA(long str ptr long long long)
 @ stdcall EnumResourceNamesExW(long wstr ptr long long long)
+@ stdcall EnumResourceNamesW(long wstr ptr long)
```

Comment:
Wouldn't this EnumResourceNamesW and FindResourceW additions, better be moved in either the "[Kernel32] sync resource.c to wine-10.0" or the "[SDK][KERNELBASE] Build loader.c - resource sync" commit?